### ISSUE TITLE
ebay-notice - Accessibility Issue

### ISSUE THREAD

**Issue by malughanshyam** [eBay/ebayui-core#1092]
# Bug Report

## Description

Accessibility issue flagged in the WAE Report. Look at the screenshot below. 
```
empty-heading
Headings must not be empty

<h3 class="page-notice__status" id="s0-0-0-24-17-6-3-0-status"><svg aria-hidden="true" class="icon icon--information-filled" xmlns="http://www.w3.org/2000/svg" focusable="false" id="s0-0-0-24-17-6-3-0-5"><use xlink:href="#icon-information-filled"></use></svg></h3>

To solve this violation, you need to:
Fix at least one (1) of these issues:
The element does not have text that is visible to screen readers
```

## Screenshots

![image](https://user-images.githubusercontent.com/5460329/78314753-baf81080-750f-11ea-87eb-bf25c411c09e.png)

![image](https://user-images.githubusercontent.com/5460329/78314848-17f3c680-7510-11ea-8773-b34ee69feb91.png)

**Comment by ianmcburnie** [comment]
Please use the `a11y-heading-text` attribute.

We will revisit this in v6 to see if we can add developer hints or warnings.

Closing ticket.
